backends/platform/sdl/win32/win32.mk

```diff
@@ -36,6 +37,7 @@ win32-data: all
 	cp $(srcdir)/doc/es/InicioRapido $(WIN32PATH)/doc/es/InicioRapido.txt
 	cp $(srcdir)/doc/fr/DemarrageRapide $(WIN32PATH)/doc/fr/DemarrageRapide.txt
 	cp $(srcdir)/doc/it/GuidaRapida $(WIN32PATH)/doc/it/GuidaRapida.txt
+	cp $(srcdir)/doc/ko/시작하기 $(WIN32PATH)/doc/ko/시작하기.txt
```

Comment:
Same thing, I am not sure it will work reliably with the name

Reply:
This file will only be used in the Korean version of WINDOWS, so I doubt there is a portability issue. It is UTF-8 code.